scripts/Packages/Package_DFN_QFN/size_definitions/dfn.yaml

```diff
@@ -181,6 +181,45 @@ MLF-6-1EP_1.6x1.6mm_P0.5mm_EP0.5x1.26mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-8-1EP_2x2mm_P0.5mm_EP0.8x1.2mm:
+  device_type: 'DFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.onsemi.com/pub/Collateral/NB3N551-D.PDF (page 7)'
+  # ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    nominal: 2
+    tolerance: 0.1
+  body_size_y:
+    nominal: 2
+    tolerance: 0.1
+  lead_width:
+    nominal: 0.25
+    tolerance: 0.05
+  lead_len:
+    nominal: 0.3
+    tolerance: 0.05
+
+  EP_size_x:
+    nominal: 0.8
+    tolerance: 0.1
+  EP_size_y:
+    nominal: 1.2
+    tolerance: 0.1
```

Comment:
The manufacturer gives a suggested size for the pad. Use the EP_size_x(y)_overwrite parameters to set it. (See http://kicad-pcb.org/libraries/klc/F6.3/ Section "Size of the EP")

Additionally the datasheet does only give min/max values not nominal plus tolerance.

```yaml
EP_size_x:
  minimum: 0.7
  maximum: 0.9
EP_size_x_overwrite: 0.9
EP_size_y:
  minimum: 1.1
  maximum: 1.3
EP_size_y_overwrite: 1.3
```